blockchain/blocks/Cargo.toml

```diff
@@ -26,6 +26,7 @@ lazy_static       = "1.4.0"
 log               = "0.4.8"
 num-bigint        = "0.4"
 once_cell         = "1.5.2"
+quickcheck        = { version = "1" }
```

Comment:
This is a test-only dependency so it should go into dev dependencies and the `Arbitrary` implementation should be hidden under `#[cfg(test)]`. Unless it's not possible for some reason?

Reply:
The `Arbitrary` implementations are spread out over several crates, unfortunately. I tried making the dependencies optional, thinking that they would be enabled when running tests but that doesn't work. We could make the dependencies optional but enabled by default but I don't think it is worth it (at least for now).